airflow/cli/commands/db_command.py

```diff
@@ -94,3 +95,26 @@ def shell(args):
 def check(_):
     """Runs a check command that checks if db is available."""
     db.check()
+
+
+@cli_utils.action_cli(check_db=False)
+def clean(args):
+    """Upgrades the metadata database"""
+    print(f"DB: {settings.engine.url!r}")
+    db.upgradedb()
```

Comment:
This seems to be some mistake. The function name is `clean` but docstring and code is about upgrading DB,

Reply:
I don't think this is even used anywhere, let's remove this function

Reply:
yup removed thanks